_docs/reference/release-roadmap.md

```diff
@@ -12,7 +12,7 @@ type: markdown
 
 Istio 0.2 will be our second release, and will work across multiple Kubernetes clusters and some basic hybrid deployments. Istio 0.2 is focused on expanding the coverage of Istio to more places, and making it easier for users and vendors to integrate into the system. Istio 0.2 will include the following features:
 - TCP telemetry collection and policy enforcement.
-- Hybrid deployments (multiple clusters, VMs).
+- Initial Raw VM support.
```

Comment:
the term Raw VM doesn't rhyme. Our discussion in istio groups literally talked about integrating with a VM cluster. Its still hybrid. The roadmap is a high level doc and we don't have to be super specific.
Fyi, @GregHanson and @kimikowang are still doing Consul integration which would make Istio work on Docker/VM only cluster. So, i am not sure why this is being kicked out, without having any discussion.